detector/list/list.go

```diff
@@ -43,7 +44,10 @@ type InitFn func() detector.Detector
 type InitMap map[string][]InitFn
 
 // CIS scanning related detectors.
-var CIS = InitMap{etcpasswdpermissions.Name: {etcpasswdpermissions.New}}
+var CIS = InitMap{
```

Comment:
This isn't related to CIS scanning (CIS being security benchmarks from https://www.cisecurity.org)

I'm not sure if there's a good existing category for this so I'd just put this directly into All.